api/v1alpha3/conversion.go

```diff
@@ -158,6 +158,10 @@ func (dst *MachineDeployment) ConvertFrom(srcRaw conversion.Hub) error {
 	return nil
 }
 
+func Convert_v1alpha4_MachineDeploymentStatus_To_v1alpha3_MachineDeploymentStatus(in *v1alpha4.MachineDeploymentStatus, out *MachineDeploymentStatus, s apiconversion.Scope) error {
+	return autoConvert_v1alpha4_MachineDeploymentStatus_To_v1alpha3_MachineDeploymentStatus(in, out, s)
```

Comment:
We usually add a comment explaining why a custom conversion is required.
In this case it should be something like
// Status.Conditions was introduced in v1alpha4, thus requiring a custom conversion function; the values is going to be preserved in an annotation thus allowing roundtrip without loosing informations

Reply:
done.